program/src/instructions/freeze_whitelist_v2.rs

```diff
@@ -7,6 +7,7 @@ use crate::{state::WhitelistV2, State};
 pub struct FreezeWhitelistV2<'info> {
     pub freeze_authority: Signer<'info>,
 
+    // TODO: missing seeds?
```

Comment:
As long as it's a:

* Valid whitelist v2
* With a freeze authority that is a signer

can be frozen. Shouldn't need the overhead of seeds checks here because the account already exists.